xml/System.Reflection/GenericParameterAttributes.xml

```diff
@@ -103,7 +103,7 @@
       </ReturnValue>
       <MemberValue>32</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The generic type parameter can be ByRefLike.</summary>
```

Comment:
I am unsure if this is a reserved keyword, but we could wrap it to highlight it properly:
```suggestion
        <summary>The generic type parameter can be <c>ByRefLike</c>.</summary>
```